tests/transport/spdp/spdp_transport.cpp

```diff
@@ -28,6 +28,14 @@
 
 #include <exception>
 #include <iostream>
+#include <string>
+
+#if defined _WIN64 || defined _WIN32
+#include <winsock.h>
+#else
+#include <arpa/inet.h>
+#include <unistd.h>
+#endif
```

Comment:
This block should be removed.  Determine which platform functions/macros you need and call the equivalent from ACE - is it just `ntohl`?